src/core/rangeset.js

```diff
@@ -45,7 +45,7 @@ class RangeSet {
     const rangeStrings = rangesString.split(',');
     _.forEach(rangeStrings, rangeString => {
       const range = rangeString.split('-').map(Number);
-      this.addRange(range[0], range[1]);
+      this.addRange(range[0], range[1] || range[0]);
```

Comment:
This would fail if we ever get bounds checking. It might be more clear to branch on `range.length`.